### PR TITLE
Create vsto.yml

### PR DESCRIPTION
**Pull request by wand3rlust**
Added a new exploitation method to leverage Office Documents.

### CODE REVIEW

File: yml/Desktop_App/vtso.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    Category: Phishing
+    Privileges: User interaction/User
+    Limitations: A user with valid privileges to approve Add-in installation. 
+    MitreID: T1204.002
+  - Steps: Add another "Add-In" to this document and include the code for any of the persistence techniques. After successful installation, persistence will be achieved.
+    Description: Setting up persistence using VTSO
+    Usecase: Persistence
+    Category: Persistence
+    Privileges: User interaction/User
+    Limitations: A user with valid privileges to approve Add-in installation and persistence mechanism. Additionally security software installed on system can also detect and block the persistence mechanism.
```
Comment (johnjhacking) [inline yml/Desktop_App/vtso.yml]: Modify to:
Limitations: A user with valid privileges to approve Add-**I**n installation and persistence mechanisms. **Additional** security software installed on system can also detect and block the persistence mechanism.

File: yml/Desktop_App/vtso.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+Name: Microsoft Visual Studio Tools for Office (VSTO)
+Description: VSTO is a functionality available in Visual Studio which allows the user to craft a .NET Office Add-in to be shipped with an office document.
```
Comment (johnjhacking) [inline yml/Desktop_App/vtso.yml]: Modify to:
VSTO is functionality available in Visual Studio, which allows the user to craft a .NET Office Add-in to be shipped with an office document.

File: yml/Desktop_App/vtso.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Created: 2023-02-10
+Usage:
+  - Steps: Create a VTSO document (Word, Powerpoint, etc) embedded with a DLL file written in .NET and send it to any user. Once the user opens the documents and approves the "Add-In" installation, code will be executed.
+    Description: Steps to exploitation using VTSO
+    Usecase: Initial Access to the system
+    Category: Phishing
+    Privileges: User interaction/User
+    Limitations: A user with valid privileges to approve Add-in installation. 
+    MitreID: T1204.002
+  - Steps: Add another "Add-In" to this document and include the code for any of the persistence techniques. After successful installation, persistence will be achieved.
```
Comment (johnjhacking) [inline yml/Desktop_App/vtso.yml]: Modifications recommended:
Utilize a different VSTO payload "Add-In" and include the code for any of the persistence techniques. After successful installation, persistence will be achieved. For more information, see "VSTO-POC" in references.

**Comment by johnjhacking** [comment]
I've placed some recommendations before i'll approve the PR. In addition, i'd appreciate if you could add whitelisting to this template, but if you don't want to lmk and i'll make it. Thanks bro! We <3 VSTO.

File: yml/Desktop_App/vtso.yml (hunk 1 above)
Comment (wand3rlust) [reply yml/Desktop_App/vtso.yml]: Done

File: yml/Desktop_App/vtso.yml (hunk 2 above)
Comment (wand3rlust) [reply yml/Desktop_App/vtso.yml]: Done

File: yml/Desktop_App/vtso.yml (hunk 3 above)
Comment (wand3rlust) [reply yml/Desktop_App/vtso.yml]: Done

**Comment by johnjhacking** [comment]
Error: ktop_App/vsto.yml:12:78: [error] trailing spaces (trailing-spaces)
Error: ktop_App/vsto.yml:29:4: [error] trailing spaces (trailing-spaces)
Error: Process completed with exit code 1.

Remove the space from the end of line 12 and 29

**Comment by wand3rlust** [comment]
> Error: ktop_App/vsto.yml:12:78: [error] trailing spaces (trailing-spaces) Error: ktop_App/vsto.yml:29:4: [error] trailing spaces (trailing-spaces) Error: Process completed with exit code 1.
> 
> Remove the space from the end of line 12 and 29

Done

**Review by johnjhacking — APPROVED** [review]

**Pull request merged**